src/Install/Console/InstallCommand.php

```diff
@@ -138,6 +138,7 @@ public function setDataSource(DataProviderInterface $dataSource)
     protected function install()
     {
         try {
+            $this->debug = $this->dataSource->isDebugMode();
```

Comment:
Can we pass this to `$this->storeConfiguration()` instead of storing it as an instance variable? You can do this on master, if you agree.

Reply:
Let's refactor that command as a whole after merging 👍